### ISSUE TITLE
Ghost effect does not remain when dragging

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-gui#1894]
### Expected Behavior

Ghost effect does not remain when dragging

### Actual Behavior

Ghost effect does not remain, although all other graphic effects do

### Steps to Reproduce

Set ghost effect on the cat to non-zero
Drag the cat
![ghostdrag](https://user-images.githubusercontent.com/2855464/39375344-0bbcd094-4a1c-11e8-91fa-e1d155732955.gif)

### Operating System and Browser
Windows 10 Chrome

**Comment by paulkaplan** [comment]
Interesting! @cwillisf i'm actually surprised, do you know why extract drawable seems to unset the ghost effect?

**Comment by fsih** [comment]
It's been explicitly disabled in the code, so maybe it's intentional?

**Comment by cwillisf** [comment]
Yeah, I believe this is intentional so that you can see the item you're dragging. Unlike other effects where the thing you're dragging might look strange due to the effects, the ghost effect can lead to deceptive behavior that seems buggy.

For example, if you have a sprite with ghost=99 stacked on top of another sprite and you try to drag the visible sprite, it can seem like a bug when the sprite you see doesn't move. In this scenario you'll accidentally drag the nearly-invisible sprite, which could also be a minor disruption of your project.

**Comment by owenkaplinsky** [comment]
Maybe dragged sprites could have some sort of additional outline, glow, or shadow (to a higher extent than it currently is), to signify that you are dragging something, while allowing for the ghost effect to persist
